Doc/tools/touch-clean-files.py

```diff
@@ -3,8 +3,10 @@
 Touch files that must pass Sphinx nit-picky mode
 so they are rebuilt and we can catch regressions.
 """
-
+import argparse
+import csv
 from pathlib import Path
+from sys import exit as exit_with_return_code_of
```

Comment:
Thanks! 

Let's keep this simple:
```suggestion
import csv
import sys
from pathlib import Path
```